lib/utilities/pkce.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:convert';
+
+import 'package:crypto/crypto.dart';
+
+import 'utils.dart' as utils;
+
+class PKCE {
```

Comment:
nit: it is named "generators" in other sdks

Reply:
hmm intend to keep PCKE as a self-wrapped modal. Consider assembling those to auth util Will recap after filling out other missing parts.